packages/backend/src/config/app.ts

```diff
@@ -53,6 +58,7 @@ export const Config = {
     ? Buffer.from(process.env.STREAM_SECRET, 'base64')
     : crypto.randomBytes(32),
 
+  useTigerbeetle: envBool('USE_TIGERBEETLE', true),
```

Comment:
:thinking: would it be worth supporting a sanity checking mode where both are used?

Reply:
How would that work? Interesting idea, we could maybe turn it into a follow-up issue?

Reply:
Put a check in each function of the "base" account service directing to the correct db service (instead of just setting up a particular account service based on `useTigerbeetle` in index.ts)
If it's both, check the results and blow up if they don't match.

I wonder if that'd even let you reduce the two db-specific account services to more direct db interfaces (instead of the full account service). Like just have `accounts.ts` and `transfers.ts` for each of them.

Reply:
https://github.com/interledger/rafiki/issues/1105